main.php

```diff
@@ -13,8 +13,11 @@
 main();
 function main()
 {
-    $hero = UnitBuilder::buildHero("Elf", "Basic", "Neil", 1);
+    //initialise the hero and a sample unit
+    $hero = UnitBuilder::buildHero("Elf", "Physical", "Neil", 1);
```

Comment:
for string constant, you can create variable.
```
const heroType = 'Elf';
const damageType = 'Physical';
const name = 'Neil';
```
retrieve using
```
self::heroType
```
```
$hero = UnitBuilder::buildHero(self::heroType, self::damageType, self::name, 1);
```

Reply:
Okay sir, will do.